db/schema.rb

```diff
@@ -30,17 +33,10 @@
     t.string "name"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer "ingredient_id"
-    t.integer "menu_id"
-    t.integer "temporary_menu_id"
-    t.index ["ingredient_id"], name: "index_dishes_on_ingredient_id"
-    t.index ["menu_id"], name: "index_dishes_on_menu_id"
-    t.index ["temporary_menu_id"], name: "index_dishes_on_temporary_menu_id"
   end
 
   create_table "ingredients", force: :cascade do |t|
     t.string "name"
-    t.string "portion_size"
```

Comment:
It seems to me those should go to the dish table? If you look at the spreadsheet, those are the properties of each dish (even though I still don't quite get what each of those means).